qemu/tests/smbios_table.py

```diff
@@ -90,13 +95,12 @@ def run(test, params, env):
                     smbios_set_para = params.object_params(sm_type).get(key,
                                                                         default_key_para)
                 else:
-                    key_index = support_machine_types.index(m_type)
-                    smbios_set_para = expect_system_versions[key_index]
```

Comment:
Seems we have the exception with the rhel 6 downstream.
```c
// @hw/pc.c
static void rhel_common_init(const char *type1_version,
                             int legacy_smbios_vendor)
{
    char buf[32];
    ...
    snprintf(buf, sizeof(buf), type1_version);
    smbios_add_field(1, offsetof(struct smbios_type_1, version_str),
                     strlen(buf) + 1, buf);
    ...
}

...

static void pc_init_rhel660(ram_addr_t ram_size,
			    const char *boot_device,
			    const char *kernel_filename,
			    const char *kernel_cmdline,
			    const char *initrd_filename,
			    const char *cpu_model)
{
	rhel_common_init("RHEL 6.6.0 PC", 0);
        ...
}
```

So I'd suggest to let `smbios_set_para` be `[m_type, expect_system_versions[key_index]]` or something else.

Reply:
Hmm, this becomes very tricky, due to the fact that 6.6.0 is very old. I think your suggestion should work. Let me take a look...